eng/pipelines/mgmt.yml

```diff
@@ -6,12 +6,21 @@ resources:
       ref: refs/tags/azure-sdk-build-tools_20211215.1
 
 parameters: 
-- name: ShouldPublish
+- name: ShouldPublishToNuget
+  type: boolean
+  default: false
+- name: ShouldPublishToDevOps
   type: boolean
   default: false
 - name: Scope
   type: string
   default: ''
+- name: DevOpsFeedID
```

Comment:
Lets not give this as an option. We should instead do something similar to https://github.com/Azure/azure-sdk-for-net/blob/main/eng/pipelines/templates/stages/archetype-net-release.yml#L263-L273 in the publishing step.